### PR TITLE
[pt-br] Add /docs/reference/glossary/kubeadm.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term Kubeadm.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/kubeadm.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka: 
+tags:
+- tool
+- operation
+---
+ Uma ferramenta para instalar rapidamente o Kubernetes e configurar um cluster seguro.
+
+<!--more--> 
+
+Você pode usar o kubeadm para instalar a camada de gerenciamento e os componentes {{< glossary_tooltip text="nós de processamento" term_id="node" >}}.
```
Comment (devlware) [inline content/pt-br/docs/reference/glossary/kubeadm.md:18]: I know that the original doc does not have a glossary_tip for control plane, what do you think about adding it here?

Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/kubeadm.md:18]: Can I add? I will do it. Thank you.

**Pull request merged**